resources/tentative/react-stockcharts/src/index.js

```diff
@@ -38,3 +39,5 @@ render(
 	<ChartComponent />,
 	document.getElementById("root")
 );
+
+window.ReactAct = act;
```

Comment:
The workload has to expose its `act` function so that we can use it from the runner.